libnavigation-ui/src/test/java/com/mapbox/navigation/ui/route/MapRouteLineTest.kt

```diff
@@ -1596,7 +1596,7 @@ class MapRouteLineTest {
             false,
             false,
             mapRouteSourceProvider,
-            0f,
+            0.0,
```

Comment:
Fixes conflict from https://github.com/mapbox/mapbox-navigation-android/pull/3661 and https://github.com/mapbox/mapbox-navigation-android/pull/3664